network.go

```diff
@@ -91,7 +90,8 @@ type DNSInfo struct {
 // NetworkInfo gathers all information related to a network interface.
 // It can be used to store the description of the underlying network.
 type NetworkInfo struct {
-	Iface  NetIfaceAddrs
+	Iface  netlink.LinkAttrs
```

Comment:
As before, can we just store the link vs LinkAttrs. You can always get LinkAttrs from Link

Reply:
Done